src/modules/grubcfg/main.py

```diff
@@ -33,11 +33,11 @@ def modify_grub_default(partitions, root_mount_point, distributor):
     default_dir = os.path.join(root_mount_point, "etc/default")
     default_grub = os.path.join(default_dir, "grub")
     distributor_replace = distributor.replace("'", "'\\''")
-    plymouth_bin = os.path.join(root_mount_point, "usr/bin/plymouth")
+    plymouth_bin = libcalamares.utils.chroot_call("sh -c \"which plymouth\"")
```

Comment:
Unless you don't even need the path further on? Were you actually looking to just get an exit status?